ui/src/app/icons/edit/component.tsx

```diff
@@ -0,0 +1,77 @@
+import {Component} from '@angular/core';
+
+import * as React from 'react';
+
+import {ReactWrapperBase} from 'app/utils';
+
+interface EditComponentState { style: object; }
+
+const style = {
+  height: 14,
+  width: 14,
+};
+
+const fillStyle = {
+  fill: '#4D4CA5'
```

Comment:
FYI, a nice trick for coloring SVGs is to set `fill='currentColor`' on the svg, which makes it inherit the current `color` value from wherever it's rendered. That way you can style it from outside by e.g. setting the color on the container.

Reply:
This specific style is only supposed to take effect on hover as per the original code, even doesn't seem to do anything visually. I still need to run this by Lou to make sure something wasn't lost at some point.